### ISSUE TITLE
Different punctuation in Continuous ASR and Continuous Translation

### ISSUE THREAD

**Issue by TakamotoAI** [Azure-Samples/cognitive-services-speech-sdk#1167]
In your complete Demo for JavaScript I noticed a different behaviour for automatic punctuation:

In Continuous ASR, the punctuation is set only after onRecognized, i.e. when a segment is finalized. 
In Continuous Translation, the punctuation of the ASR is set in real-time while onRecognizing, i.e. while hypotheses are sent back.

It is possible to have real-time punctuation also in Continuous ASR without having to wait until a segment is finalized? The real-time punctuation provided with Continuous Translation seems good enough.

**Comment by glharper** [comment]
@TakamotoAI Would `config.SetProperty(PropertyId.SpeechServiceResponse_PostProcessingOption, "TrueText");` be what you're looking for? Thanks for using Speech SDK, and writing this issue up.

**Comment by pankopon** [comment]
Closed since no further comments in 20 days. Please open a new issue if more support is needed.
